package.json

```diff
@@ -40,6 +40,7 @@
     "@google-cloud/common": "^0.11.0",
     "continuation-local-storage": "^3.1.4",
     "gcp-metadata": "^0.1.0",
+    "is": "^3.2.0",
```

Comment:
We `lodash` for the `isObject` check in other libraries and `is` uses a different `isObject` check than `lodash`. It would be good to have only one notion of what an object is.